cmake/Modules/FindAzureSDK_EP.cmake

```diff
@@ -83,38 +85,88 @@ if (NOT AZURESDK_FOUND)
     endif()
 
     if (WIN32)
-      set(CFLAGS_DEF "${CMAKE_C_FLAGS}")
-      set(CXXFLAGS_DEF "${CMAKE_CXX_FLAGS}")
+      if(MSVC)
+        set(CFLAGS_DEF " -I${TILEDB_EP_INSTALL_PREFIX}/include /Dazure_storage_lite_EXPORTS /DCURL_STATICLIB=1 ${CMAKE_C_FLAGS}")
+        set(CXXFLAGS_DEF " -I${TILEDB_EP_INSTALL_PREFIX}/include /Dazure_storage_lite_EXPORTS /DCURL_STATICLIB=1 ${CMAKE_CXX_FLAGS}")
+      endif()
+      set(CURL_INCLUDE_DIR "${TILEDB_EP_INSTALL_PREFIX}/include")
     else()
-      set(CFLAGS_DEF "${CMAKE_C_FLAGS} -fPIC")
-      set(CXXFLAGS_DEF "${CMAKE_CXX_FLAGS} -fPIC")
+      #put our switch first in case other items are empty, leaving problematic blank space at beginning
+      set(CFLAGS_DEF "-fPIC ${CMAKE_C_FLAGS}")
+      set(CXXFLAGS_DEF "-fPIC ${CMAKE_CXX_FLAGS}")
     endif()
-
-    ExternalProject_Add(ep_azuresdk
-      PREFIX "externals"
-      # Set download name to avoid collisions with only the version number in the filename
-      DOWNLOAD_NAME ep_azuresdk.zip
-      URL "https://github.com/Azure/azure-storage-cpplite/archive/v0.2.0.zip"
-      URL_HASH SHA1=058975ccac9b60b522c9f7fd044a3d2aaec9f893
-      CMAKE_ARGS
-        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-        -DBUILD_SHARED_LIBS=OFF
-        -DBUILD_TESTS=OFF
-        -DBUILD_SAMPLES=OFF
-        -DCMAKE_PREFIX_PATH=${TILEDB_EP_INSTALL_PREFIX}
-        -DCMAKE_INSTALL_PREFIX=${TILEDB_EP_INSTALL_PREFIX}
-        -DCMAKE_CXX_FLAGS=-fPIC
-        -DCMAKE_C_FLAGS=-fPIC
-      PATCH_COMMAND
-        patch -N -p1 < ${TILEDB_CMAKE_INPUTS_DIR}/patches/ep_azuresdk/remove-uuid-dep.patch &&
-        patch -N -p1 < ${TILEDB_CMAKE_INPUTS_DIR}/patches/ep_azuresdk/azurite-support.patch
-      LOG_DOWNLOAD TRUE
-      LOG_CONFIGURE TRUE
-      LOG_BUILD TRUE
-      LOG_INSTALL TRUE
-      LOG_OUTPUT_ON_FAILURE ${TILEDB_LOG_OUTPUT_ON_FAILURE}
-      DEPENDS ${DEPENDS}
+    
+    if (WIN32)
+        # needed for applying patches on windows
+        find_package(Git REQUIRED)
+        #see comment on this answer - https://stackoverflow.com/a/45698220
+        #and this - https://stackoverflow.com/a/62967602 (same thread, different answer/comment)
+    else()
+        find_package(Git REQUIRED)
```

Comment:
Git should not be required for non-windows platforms. This will break a large number of non-windows build environments that we have today.

Reply:
Changed back to using patch in non-win32 path.

Reply:
Thanks for the adjustment!